### ISSUE TITLE
bias

### ISSUE THREAD

**Issue by hbw945** [DingXiaoH/RepLKNet-pytorch#30]
Why should bias be set to false，thank you！

**Comment by DingXiaoH** [comment]
This is a common practice when BatchNorm follows conv.
